thrift/src/main/thrift/atoms/interactive.thrift

```diff
@@ -0,0 +1,13 @@
+namespace * contentatom.cta
+namespace java com.gu.contentatom.thrift.atom.interactive
+#@namespace scala com.gu.contentatom.thrift.atom.interactive
+
+struct InteractiveAtom {
+  /* the unique ID will be stored in the `atom` data*/
+  2: required string type
```

Comment:
Not sure, I copied the structure used for the other atom types (e.g. https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/atoms/explainer.thrift)